components/GraphDiagram/GraphDiagram.scss

```diff
@@ -52,7 +52,8 @@
     line-height: 1em;
 
     &.selected {
-      background-color: var(--highlight);
+      background-color: var(--accent);
+      color: var(--text-on-accent);
```

Comment:
I like the highlight->accent name change. 💯 

Did you intend to lose the `warning` style for deprecated modules when selected...
before:
![CleanShot 2024-07-21 at 10 16 41](https://github.com/user-attachments/assets/f467b6b5-56ca-4160-b346-213d4dc1d571)

after:
![CleanShot 2024-07-21 at 10 18 24](https://github.com/user-attachments/assets/53112290-036c-4f87-95da-1fa8d17b796f)

Also, the white-on-orange selection text is a little harder to read.  Not a lot, but it's enough that I notice the difference with my eyesight.  I know we have this style elsewhere in the UI (tabs, zoom buttons), but in those places the text is larger and not as dynamic.

Reply:
Dejavu, pretty sure I made the same exact change a few years ago. I don’t think it's a huge issue honestly, because the selected item is shown in the sidebar, which is even more explicit in telling you what that color means:

<img width="846" alt="Screenshot 7" src="https://github.com/user-attachments/assets/2a81be95-2cf2-4afa-9622-4ab731dd652c">